cmd/world/cardinal/dev.go

```diff
@@ -202,8 +208,8 @@ func runRedis() error {
 
 // runCardinalPrep preparation for runs cardinal in dev mode.
 // We run cardinal without docker to make it easier to debug and skip the docker image build step
-func runCardinalPrep() error {
-	err := os.Chdir("cardinal")
+func runCardinalPrep(rootDir string) error {
+	err := os.Chdir(filepath.Join(rootDir, "cardinal"))
```

Comment:
The [WORLD-1078](https://linear.app/arguslabs/issue/WORLD-1078/world-cardinal-dev-has-the-cardinal-directory-hard-coded) request was to make this "cardinal" variable configurable. 

The use case is the `world-engine/e2e/testgames` directory. It would be nice to be able to run
`world cardinal dev` from that directory and have it find and run the code in `game`. Currently there's a symlink from cardinal->game. I'd like to get rid of the symlink.

Reply:
ohh got it. but i don't think `root_dir` can be used as cardinal dir path. because `root_dir` is the path forw `world.toml` file.

Maybe we can add new field like `game_dir` in `world.toml` for that.

Reply:
done. added `game_dir` on the `world.toml`